scripts/e2e.sh

```diff
@@ -32,7 +32,8 @@ echo "e2e_tower_username: '"$TOWER_USERNAME"'" >> e2e_test_config.yml
 echo "Bootstrapping tower in openshift project: ${TOWER_OPENSHIFT_PROJECT}"
 ansible-playbook -i ./inventories/hosts \
 playbooks/bootstrap.yml \
---extra-vars "@e2e_test_config.yml" 
+--extra-vars "@e2e_test_config.yml" \
+-e cicd=true
```

Comment:
@StevenTobin this is already set in the config file. It shouldn't be needed here but I don't think it would cause any hassle either. Just fyi.

Reply:
Ah yeah, I saw it appear on a rebase and forgot about the config file so added it back.

I'll remove it as it's unneccessary.